labman/db/tests/test_process.py

```diff
@@ -1694,6 +1771,2312 @@ def test_generate_prep_information(self):
         exp = {Study(1): SHOTGUN_EXAMPLE}
         self.assertEqual(obs[Study(1)], exp[Study(1)])
 
+
+INTERLEAVED_POSITIONS = [
```

Comment:
This list is magical, would it be possible to get a comment to for some context? The ordering of the indices seems to map back to the docstring in `get_interleaved_quarters_position_generator`, but I may be wrong on that. If the ordering does correspond to that docstring, than perhaps just a comment to the effect of "See the docstring for get_interleaved_quarters_position_generator for more information" may be sufficient.

Reply:
I've added an explanatory comment.